src/components/EmptyState/EmptyState.tsx

```diff
@@ -69,6 +72,12 @@ export default class EmptyState extends React.PureComponent<Props, never> {
       ? buttonFrom(secondaryAction, {plain: true})
       : null;
 
+    const footerMarkup = footer ? (
+      <div className={styles.Footer}>
+        <TextContainer>{footer}</TextContainer>
```

Comment:
Since the `footer` prop will accept any `React.ReactNode` I wouldn't add wrap it in a `TextContainer` as we won't know what the children are (the consumer can be responsible for spacing their markup with a `Stack` or `TextContainer` etc)

Reply:
I was trying to keep `footer` as similar as possible to `children` (`React.ReactNode` in `TextContainer`), since the idea of a footer is sort of this additional context.

Reply:
Ahh gotcha, in that case we should think about naming the prop something like `footerContent` or something that will make it clear that this should be text based 👍